ntirpc/rpc/clnt.h

```diff
@@ -115,7 +109,7 @@ typedef struct rpc_client {
 
 	mutex_t cl_lock;	/* serialize private data */
 	uint32_t cl_refcnt;	/* handle refcnt */
-	uint32_t cl_flags;	/* state flags */
+	uint16_t cl_flags;	/* state flags */
```

Comment:
So, this doesn't actually save any space, since the compiler wil pad it to 4 bytes.  It's probably not worth the change.